### DISCUSSION TITLE
dukun langit/Roman for sky cotl not working

### DISCUSSION THREAD

**Discussion by LyomaBeans** [thatskymod/Sky-CotL-Scripts#98] [Q&A]
I went to do daily cr as i am really busy and will be out of town but whenever i try to auto run candles or give or receive light i just keep getting and error message, and for cr it is just stuck and 1% i tried reinstalling canvas, redownloading mods but nothing seems to work. is it because of the update for sky? when will it be working again?

**Comment by Grapes9** [comment]
same and evertime i click teleport blink my game will crash

**Reply by LyomaBeans** [reply]
yeah that to everything seems to be crashing games or just not working and it sucks because events are happening and im not able to candle run for em ):

**Reply by thatskymod** [reply]
@ObservantLurker Thanks for clarifying. Username checks out!

**Comment by thatskymod** [comment]
The affected mods, including [Dukun Langit](https://github.com/thatskymod/Sky-CotL-Scripts/blob/23a64c1146d825038426f74acdd1909aa779674b/Canvas/Libs/libLangit034.so) and [libRoman](https://github.com/thatskymod/Sky-CotL-Scripts/blob/23a64c1146d825038426f74acdd1909aa779674b/Canvas/Libs/libRomanBeta.so), have been updated to their newest versions and should now work properly again.

thatskymod
Repository maintainer
